bridge/checkpoint/checkpoint.go

```diff
@@ -117,9 +122,12 @@ func (c *checkpoint) StartNewCheckpoint(epochSize uint64) error {
 	return nil
 }
 
-func (c *checkpoint) determineCheckpointRange(lastChildBlock, epochSize uint64) (uint64, uint64) {
-	// TODO: implement
-	return lastChildBlock + 1, lastChildBlock + epochSize
+func (c *checkpoint) determineCheckpointRange(lastChildBlock uint64) (uint64, uint64) {
```

Comment:
What do you think about changing the method signature to:
`func (c *checkpoint) determineCheckpointRange(lastChildBlock uint64) (start uint64, end uint64)`

and then just reference these values inside before the return? I think it's much clear that way what the return values are, especially because the method doesn't have any kind of description.

Reply:
Thank you for suggestion, fixed b65283c9249dc562ea3597d3db2fdf69c78213fd

Reply:
Thank you for suggestion, fixed https://github.com/0xPolygon/polygon-edge/commit/b65283c9249dc562ea3597d3db2fdf69c78213fd